python_spike/case_ref_debugging.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+from caseprocessor.messaging.new_case_receiver import __case_ref_generator_key
```

Comment:
I know it's not part of this change, but we should avoid using double underscore names in almost all circumstances apart from when we want it's name mangling behaviour. To indicate an object is intended to be private then we can use single underscore names.